docs/source/concepts/ledger-model/ledger-privacy.rst

```diff
@@ -85,18 +91,18 @@ for, providing privacy to `A` and `P` with respect to the bank.
 
 .. _def-informee:
 
-As a design choice, Daml Ledgers show to observers on a contract only the
+As a design choice, Daml Ledgers show to contract observers only the
 :ref:`state changing <def-contract-state>` actions on the contract.
-More precisely, **Fetch** and non-consuming **Exercise** actions are not shown to the observers - except when they are
-the actors of these actions.
+More precisely, **Fetch** and non-consuming **Exercise** actions are not shown to contract observers - except when they are
+also actors or choice observers of these actions.
 This motivates the following definition: a party `p` is an **informee** of an action `A` if one of the following holds:
 
   * `A` is a **Create** on a contract `c` and `p` is a stakeholder of `c`.
 
-  * `A` is a consuming **Exercise** on a contract `c`, and `p` is a stakeholder of `c` or an actor on `A`.
+  * `A` is a consuming **Exercise** on a contract `c`, and `p` is a stakeholder of `c`, an actor on `A`, or a choice observer on `A`.
```

Comment:
Since the whole list is a distinction, I'd suggest just adding an item for choice observers rather than duplicating and including nested disjunctions

Reply:
That's a great idea! 👍